modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -107,7 +91,7 @@ def update_sha256(form)
     end
 
     def get_full_url(url)
-      "#{BASE_URL}/vaforms/#{url.gsub('./', '')}" if url.starts_with?('./va') || url.starts_with?('./medical')
+      "#{FORM_BASE_URL}/vaforms/#{url.gsub('./', '')}" if url.starts_with?('./va') || url.starts_with?('./medical')
```

Comment:
```suggestion
      raise ArgumentError, 'url must start with ./va or ./medical' unless url.starts_with?('./va') || url.starts_with?('./medical')

      "#{FORM_BASE_URL}/vaforms/#{url.gsub('./', '')}"
```
if this method returns a nil (which could happen as it's currently written: `x if y`), the `Addressable::URI.parse` line will raise an exception. should we just raise the exception here so we know what the problem was?

Reply:
I dont think so, the intent here is to make sure the method is used properly for these url patterns. moving the exception to the URI parse may loose the intent during future work of the class. Perhaps renaming it to something like expand_va_url would be more clear of intent

Reply:
>moving the exception to the URI parse
currently the exception is at the URI parse